README.md

```diff
@@ -88,41 +90,50 @@ class State(rx.State):
 
         self.processing, self.complete = True, False
         yield
-        response = openai.Image.create(prompt=self.prompt, n=1, size="1024x1024")
-        self.image_url = response["data"][0]["url"]
+        response = openai_client.images.generate(
+            prompt=self.prompt, n=1, size="1024x1024"
+        )
+        self.image_url = response.data[0].url
         self.processing, self.complete = False, True
-        
+
 
 def index():
     return rx.center(
         rx.vstack(
-            rx.heading("DALL·E"),
-            rx.input(placeholder="Enter a prompt", on_blur=State.set_prompt),
-            rx.button(
-                "Generate Image",
-                on_click=State.get_image,
-                is_loading=State.processing,
+            rx.heading("DALL-E", font_size="1.5em"),
+            rx.radix.text_field.root(
+                rx.input(
+                    placeholder="Enter a prompt..",
+                    on_blur=State.set_prompt,
+                    width="100%",
+                ),
                 width="100%",
             ),
+            rx.button("Generate Image", on_click=State.get_image, width="100%"),
             rx.cond(
-                State.complete,
-                     rx.image(
-                         src=State.image_url,
-                         height="25em",
-                         width="25em",
-                    )
+                State.processing,
+                rx.chakra.circular_progress(is_indeterminate=True),
+                rx.cond(
+                    State.complete,
+                    rx.image(
+                        src=State.image_url,
+                    ),
+                ),
             ),
-            padding="2em",
-            shadow="lg",
-            border_radius="lg",
+            width="25em",
+            padding="1em",
+            spacing="2",
+            align_items="center",
+            border_radius="10px",
```

Comment:
Potentially confusing to newcomers that we're mixing px, em, and % all together for. maybe we can unify for the intro app